es/cheatsheet-supervised-learning.md

```diff
@@ -1,567 +1,566 @@
 
 <br>
 
 **17. Remark: Stochastic gradient descent (SGD) is updating the parameter based on each training example, and batch gradient descent is on a batch of training examples.**
 
-&#10230;
+&#10230; Observación: El descenso del gradiente estocástico (SGD, por sus siglas en inglés) actualiza el parámetro basándose en cada ejemplo de entenamiento mientras que el descenso por lotes realiza la actualización del parámetro basándose en un conjunto (un lote) de ejemplos de entrenamiento.
 
 <br>
 
 **18. Likelihood ― The likelihood of a model L(θ) given parameters θ is used to find the optimal parameters θ through maximizing the likelihood. In practice, we use the log-likelihood ℓ(θ)=log(L(θ)) which is easier to optimize. We have:**
 
-&#10230;
+&#10230; Probabilidad ― La probabilidad de un modelo L(θ) dados los parámetros θ es utilizada para hallar los valores óptimos de  θ a través de la probabilidad. En la práctica se utiliza la log-probabilidad ℓ(θ)=log(L(θ)) la cual es fácil de optimizar. Tenemos:
 
 <br>
 
 **19. Newton's algorithm ― The Newton's algorithm is a numerical method that finds θ such that ℓ′(θ)=0. Its update rule is as follows:**
 
-&#10230;
+&#10230; Algoritmo de Newton ― El algoritmo de Newtow es un método numérico para hallar θ tal que ℓ′(θ)=0. Su regla de actualización es:
 
 <br>
 
 **20. Remark: the multidimensional generalization, also known as the Newton-Raphson method, has the following update rule:**
 
-&#10230;
+&#10230; Observación: la generalización multidimensional, también conocida como método de Newton-Raphson, tiene la siguiente regla de actualización:
 
 <br>
 
 **21. Linear models**
 
-&#10230;
+&#10230; Modelos lineales
 
 <br>
 
 **22. Linear regression**
 
-&#10230;
+&#10230; Regresión lineal
 
 <br>
 
 **23. We assume here that y|x;θ∼N(μ,σ2)**
 
-&#10230;
+&#10230; Asumimos que y|x;θ∼N(μ,σ2)
 
 <br>
 
 **24. Normal equations ― By noting X the matrix design, the value of θ that minimizes the cost function is a closed-form solution such that:**
 
-&#10230;
+&#10230; Ecuaciones normales ― Sea X la matriz de diseño, el valor de θ que minimiza la función de costo es una solución en forma cerrada tal que:
 
 <br>
 
 **25. LMS algorithm ― By noting α the learning rate, the update rule of the Least Mean Squares (LMS) algorithm for a training set of m data points, which is also known as the Widrow-Hoff learning rule, is as follows:**
 
-&#10230;
+&#10230; Algoritmo LMS ― Sea α la tasa de aprendizaje, la regla de actualización del algoritmo LMS (del inglés, Least Mean Squares) para el entrenamiendo de m puntos, conocida también como tasa de aprendizaje de Widrow-Hoff, se define como:
 
 <br>
 
 **26. Remark: the update rule is a particular case of the gradient ascent.**
 
-&#10230;
+&#10230; Observación: la regla de actualización es un caso particular del ascenso por gradiente.
```

Comment:
Again, consider to use "[_Descenso de gradiente_](https://developers.google.com/machine-learning/glossary/#gradient_descent)".